ntttcp/ntttcp.TESTGEN.ps1

```diff
@@ -36,11 +32,12 @@ function test_recv {
         [parameter(Mandatory=$true)]   [Int]    $Port,
         [parameter(Mandatory=$false)]  [string] $Proto,
         [parameter(Mandatory=$true)]   [String] $OutDir,
-        [parameter(Mandatory=$true)]   [String] $Fname
+        [parameter(Mandatory=$true)]   [String] $Fname,
+        [parameter(Mandatory=$true)]   [String] $Options
     )
 
     [string] $out = (Join-Path -Path $OutDir -ChildPath "$Fname")
-    [string] $cmd = "ntttcp.exe -r -m $Conn,*,$g_DestIp $proto -v -wu $g_ptime -cd $g_ptime -sp -p $Port -t $g_runtime -xml $out.xml"
+    [string] $cmd = "ntttcp.exe -r -m $Conn,*,$g_DestIp $proto $Options -v -wu $($g_Config.Warmup) -cd $($g_Config.Cooldown) -sp -p $Port -t $($g_Config.Time) -xml $out.xml"
```

Comment:
please rename to g_Config.Runtime, as Time is an overloaded term and user might not immediately connect it with the test runtime/duration